strings/french.strings

```diff
@@ -426,6 +426,12 @@
 /* MISSING TRANSLATION - views/pages/GroupChat.vue */
 "{numMembers} members" = "{numMembers} members";
 
+/* MISSING TRANSLATION - views/pages/Home.vue */
+"Create a new group and invite your friends." = "Create a new group and invite your friends.";
+
+/* MISSING TRANSLATION - views/pages/Home.vue */
+"Create" = "Create";
+
```

Comment:
Did you want to add translations to these?

Reply:
Sure, I am preparing another PR for these and other missing translations